src/audio3d/Audio3DConverts.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __PIROGRONIAN__AUDIO_3D__CONVERTS__H__
+#define __PIROGRONIAN__AUDIO_3D__CONVERTS__H__
+
+#include <Eigen/Core>
+#include <AL/al.h>
+
+namespace Audio3D 
+{
+
+    Eigen::Vector3d inline convert(const ALfloat *v)
```

Comment:
Vector3f?

Reply:
and you should use better name, convert is too generic

Reply:
I though about it, but was in unsure about better names. After all, this is part of small module inside its own namespace...

Reply:
actually, with `Eigen::Map<>()` and `Eigen::Vector3f::data()` you may not need this method.

Reply:
Actually, it appeared all this file is unnecessary and I removed it.